echopype/tests/echodata/test_echodata_combine.py

```diff
@@ -35,6 +33,14 @@ def ek60_reversed_ping_time_test_data(test_path):
     return [test_path["EK60"].joinpath(*f) for f in files]
 
 
+@pytest.fixture
+def ek60_diff_range_sample_test_data():
+    return ["s3://noaa-wcsd-pds/data/raw/Bell_M._Shimada/SH1701/EK60/TEST-D20170114-T202932.raw",
+            "s3://noaa-wcsd-pds/data/raw/Bell_M._Shimada/SH1701/EK60/TEST-D20170114-T203337.raw",
+            "s3://noaa-wcsd-pds/data/raw/Bell_M._Shimada/SH1701/EK60/TEST-D20170114-T203853.raw",
+            "s3://noaa-wcsd-pds/data/raw/Bell_M._Shimada/SH1701/EK60/TEST-D20170114-T204035.raw"]
```

Comment:
The files are under [./ek60/ncei-wcsd/SH1701](https://drive.google.com/drive/u/1/folders/1KwomLFfRAOVzwxkMbXD36oEKp6PPURg3)

Here I didn't reduce the files because I didn't have EK60 software, only EK80 software (so if I record they'll come out as EK80 files), but I removed the last one because the length of `range_sample` was the same as the 3rd file.